bucket/archi.json

```diff
@@ -24,7 +24,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://www.archimatetool.com/downloads/archi.php?/$version/Archi-Win64-$version.zip"
+                "url": "https://www.archimatetool.com/downloads/archi5.php?/$version/Archi-Win64-$version.zip"
```

Comment:
Like my other comment, the URL should be : 

`https://www.archimatetool.com/downloads/archi-5.php?/$version/Archi-Win64-$version.zip`

Reply:
- fixed second URL
- also renamed the PR